### ISSUE TITLE
"The detective's holobadge rejects your insufficient access rights." as a detective

### ISSUE THREAD

**Issue by nikothedude** [Skyrat-SS13/Skyrat-TG#25293]
Reporting client version: 514.1589

## Round ID:
10016

## Testmerges:
- [CIN Evilguns | The gunnenshootening pt 3 | Long live the king | Slight rework of the CIN surplus company](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24304)
- [Armor tech of the current* year | Protect yourself from all them "guncargos" the old people keep warning you about](https://github.com/Skyrat-SS13/Skyrat-tg/pull/24947)

## Reproduction:

Step 1: Get a det's holobadge as detective
Step 2: Try swiping your ID on it
Step 3: Apparantly detectives cant use detective holobadges

This seems like a major oversight.

**Comment by Namwty** [comment]
I imagine this one fixes it ( due to access_security, unless Im wrong ) and generally makes holobadges more common etc. Incase someone missed PR as it's been up for some time.

#25198

**Comment by Plum-but-gayer** [comment]
Yep! The holobadge PR will fix it. :) Thanks for reporting anyway though!! Good skyrat citizen points to you...
